include/seqan3/range/container/small_vector.hpp

```diff
@@ -315,13 +315,13 @@ class small_vector
         return &data_[0];
     }
 
-    //!\copydoc small_vector::begin().
+    //!\overload
```

Comment:
Why not both copydoc?

Reply:
well because one of them is an overload :D and `//!\overload` is safer to use because the copydoc link generation changes (one of the errors in the newest doxygen version).